ow_lander/scripts/camera_capture_action_server.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+
+import actionlib
+
+import ow_lander.msg
+from std_msgs.msg import Empty, Float64
+from sensor_msgs.msg import PointCloud2
+
+class CameraCaptureActionServer(object):
+  def __init__(self, name):
+    self._action_name = name
+    self._exposure_pub = rospy.Publisher(
+      '/gazebo/plugins/camera_sim/exposure', Float64, queue_size=10)
+    self._trigger_pub = rospy.Publisher(
+      '/StereoCamera/left/image_trigger', Empty, queue_size=10)
+    self._point_cloud_sub = rospy.Subscriber('/StereoCamera/points2',
+                                             PointCloud2, self._handle_point_cloud)
+    self._server = actionlib.SimpleActionServer(self._action_name,
+                                                ow_lander.msg.CameraCaptureAction,
+                                                execute_cb=self.execute_cb,
+                                                auto_start = False)
+    self._server.start()
+
+  def execute_cb(self, goal):
+    # helper variables
+    r = rospy.Rate(1)
+    self.point_cloud_created = False
+
+    # Set exposure if it is specified
+    if goal.exposure > 0:
+      self._exposure_pub.publish(goal.exposure)
+      # There is no guarantee that exposure is set before the image is triggered.
+      # Pause to make it highly likely that exposure is received first.
+      r.sleep()
+
+    self._trigger_pub.publish()
+
+    for i in range(0, 5):
```

Comment:
Minor/optional: could use a named timeout constant instead of '5' .

Reply:
Done. Also running at 10 Hz instead of 1 Hz now.